src/fsharp/nameres.fs

```diff
@@ -2521,6 +2533,10 @@ let ResolveCompletionsInType (ncenv: NameResolver) nenv isApplicableMeth m ad st
         |> List.filter (fun x -> not (PropInfoIsUnseen m x))
 
     let minfoFilter (minfo:MethInfo) = 
+        let isApplicableMeth =
+            match completionTargets with
+            | ResolveCompletionTargets.All isApplicableMeth -> isApplicableMeth
```

Comment:
maybe `ResolveCompletionTargets.All x -> x` is less confusing because `isApplicableMeth` is the binding name